src/tox_lsr/config_files/woke.yml

```diff
@@ -3,13 +3,15 @@ ignore_files:
   - .github
   - .sanity-ansible-ignore-*.txt
   - pylintrc
+  - vendor
```

Comment:
I'm assuming this is for the metrics role?  If so, then we need to work with @natoscott to address issues in ansible-pcp